cmd/lakectl/cmd/refs_restore.go

```diff
@@ -45,9 +46,50 @@ Since a bare repo is expected, in case of transient failure, delete the reposito
 		}
 		// execute the restore operation
 		client := getClient()
-		resp, err := client.RestoreRefsWithResponse(cmd.Context(), repoURI.Repository, apigen.RestoreRefsJSONRequestBody(manifest))
-		DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusOK)
-		Write(refsRestoreSuccess, nil)
+		ctx := cmd.Context()
+		resp, err := client.RestoreRefsSubmitWithResponse(ctx, repoURI.Repository, apigen.RestoreRefsSubmitJSONRequestBody{
+			BranchesMetaRangeId: manifest.BranchesMetaRangeId,
+			CommitsMetaRangeId:  manifest.CommitsMetaRangeId,
+			TagsMetaRangeId:     manifest.TagsMetaRangeId,
+		})
+		DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusAccepted)
+		if resp.JSON202 == nil {
+			Die("Bad response from server", 1)
+		}
+		taskID := resp.JSON202.Id
+		logging.FromContext(ctx).
+			WithField("task_id", taskID).
+			Debug("Submitted refs restore")
+
+		const checkInterval = 3 * time.Second
```

Comment:
This one is not customizable? (I don't really mind either way)

Reply:
Adding flag to customize interval check also for restore. Default move to common code.